src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -343,6 +343,60 @@ public async Task<IActionResult> CancelDisputeAsync(long disputeId, Cancellation
         }
     }
 
+    /// <summary>
+    /// An endpoint for resending an email to a Disputant.
+    /// </summary>
+    /// <param name="disputeId"></param>
+    /// <param name="host"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns>
+    /// <response code="200">OK.</response>
+    /// <response code="400">The request was not well formed. Check the parameters.</response>
+    /// <response code="401">Unauthenticated.</response>
+    /// <response code="403">Forbidden, requires dispute:submit permission.</response>
+    /// <response code="500">There was a server error that prevented the update from completing successfully.</response>
+    /// </returns>
+    [HttpPut("{disputeId}/resendEmailVerify")]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]

```

Comment:
I don't see 401 and 403 in the catch blocks of the ApiException. Are these returned? It looks like these would be converted to 500 errors.

Reply:
these would be caught before the body of the method was ever executed which is why they are not part of the other exceptions listed that might happen after the method was partway done executing.